docs/source/improve-workflows/finetuning/dpo_with_nemo_customizer.md

```
@@ -0,0 +1,950 @@
+```yaml
+hyperparameters:
+  dpo:
+    ref_policy_kl_penalty: 0.1  # Default: balanced exploration
+    # ref_policy_kl_penalty: 0.01  # Lower: more aggressive updates
+    # ref_policy_kl_penalty: 0.5   # Higher: more conservative updates
+```
+
+- **Lower values (0.01-0.05)**: Allow larger policy updates, faster learning but risk of instability
+- **Higher values (0.2-0.5)**: More conservative updates, slower but more stable training
+
+**SFT Loss Weight**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)